src/Avalonia.Visuals/Rendering/DeferredRenderer.cs

```diff
@@ -414,7 +415,11 @@ private void OnRenderLoopTick(object sender, EventArgs e)
                     Interlocked.Exchange(ref scene, _scene);
                     Render(scene);
                 }
-                catch { }
+                catch (Exception ex)
+                {
+                    Debug.Fail("Fail during render: " + ex.Message, ex.ToString());
+                    throw;
```

Comment:
You are changing behavior from swallowing exception silently to crashing the app without any way to recover.

Reply:
I discussed this with @grokys - exceptions there caused severe unrecoverable rendering issues. IMO the app should crash in those cases.